web/gds-admin-ui/src/helpers/api/auth.js

```diff
@@ -29,4 +24,8 @@ function forgotPasswordConfirm(params: any): any {
     return api.create(`${baseUrl}`, params);
 }
 
-export { login, logout, signup, forgotPassword, forgotPasswordConfirm };
+function postCredentials(credentials, params) {
+    return api.create('/authenticate', credentials, params)
+}
+
+export { login, signup, forgotPassword, forgotPasswordConfirm, postCredentials };
```

Comment:
We can probably remove `signup`, `forgotPassword`, and `forgotPasswordConfirm`. How is `login` different from `postCredentials`? We should probably make that just one function.

Reply:
Okay